src/main/java/ca/team2706/frc/robot/commands/bling/BlingController.java

```diff
@@ -0,0 +1,148 @@
+package ca.team2706.frc.robot.commands.bling;
+
+import ca.team2706.frc.robot.commands.bling.patterns.Blank;
+import ca.team2706.frc.robot.commands.bling.patterns.BlingPattern;
+import ca.team2706.frc.robot.commands.bling.patterns.TestPattern;

```

Comment:
@KyleRAnderson It looks like this class wasn't pushed, and is causing the build to fail. Do you want me to take out `TestPattern` references in my pull request, or do you want to fix it in a separate PR?

Reply:
@eandr127 I've already fixed it in master. If you pull you should be good.

TestPattern was what I used to test the bling system. I didn't commit it, but I also forgot to remove references to it, which meant on my side it looked like everything was fine.